sdk/python/README.md

```diff
@@ -1,59 +0,0 @@
-# Pulumi AWS Native Provider (preview)
```

Comment:
Not sure why this was removed?

Reply:
Hmm no idea why. Reinstated. Thanks for catching.